### ISSUE TITLE
not running on windows

### ISSUE THREAD

**Issue by legrandmawak** [jaredly/vscode-background-terminal-notifier#1]
this command requires unix based system.
ps is not a recognized command

**Comment by jaredly** [comment]
That's right. We could probably use something like https://github.com/indexzero/ps-tree or https://www.npmjs.com/package/ps-node to have cross-platform suppor

**Comment by scrthq** [comment]
Since we're using `exec`, which is in turn executing via `cmd.exe` on Windows, what about using `tasklist`, which is more or less the equivalent of `ps` on the Windows side? It could totally be missing dependent flags though that I'm not aware of. Time to spin up a new Linux VM to compare 😃

**Comment by jaredly** [comment]
that sounds great!

**Comment by DohanKim** [comment]
Just in case anyone need the feature,
https://github.com/jaredly/vscode-background-terminal-notifier/pull/8

**Comment by Maxim-Mazurok** [comment]
#8 didn't work for me, so I did #9

**Comment by Niels-c-Scholten** [comment]
Is this resolved in the latest version? I am still getting the error: "Error while listening for process end: Error: Command failed: ps -o tty 19044 'ps' is not recognized as an internal or external command...
I am running on Windows.
